src/seed_generators.rs

```diff
@@ -0,0 +1,30 @@
+use crate::felt::FeltTrait;
+extern crate chrono;
+
+pub struct MegaLehmerSeed<F: FeltTrait> {
+    seed: F,
+}
+
+impl<F: FeltTrait> MegaLehmerSeed<F> {
+    // No further manipulation just the time
```

Comment:
Let's use `///` for documentation as in: https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html#outer-and-inner-documentation